templates/zerver/help/search-for-messages.md

```diff
@@ -74,31 +130,22 @@ query using the `stream:` or `streams:` operators will search all
 messages that you have access to in the selected stream(s).  For
```

Comment:
This is the only remaining instance of "operators" on the page. Do we want to keep it or change it to "filters"?

I will note that we've recently linked to this section from the [API documentation](https://zulip.com/api/construct-narrow) as it's a helpful explanation for folks using narrows via the API, which does use the more technical term of "operator".

Reply:
Changed to "filters", even though @timabbott  might complain.

Reply:
I don't think the API docs mismatching this is an issue; nobody will be confused by it. I think `using the `stream:` or `streams:` filter type will search`  might be clearer, but I don't see anyone being confused by this as written.